codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/ServerHttpProtocolGenerator.kt

```diff
@@ -746,7 +747,7 @@ private class ServerHttpProtocolImplGenerator(
         with(writer) {
             rustTemplate(
                 """
-                let query_string = request.uri().query().ok_or(#{SmithyHttpServer}::rejection::MissingQueryString)?;
```

Comment:
Can you also remove the struct from `rust-runtime/aws-smithy-http-server/src/rejection.rs`? This is the only place where it was used.

Reply:
Sure, done